setup.cfg

```diff
@@ -98,7 +98,7 @@ azure =
     azure-identity>=1.4.0
     knack
 gdrive = pydrive2[fsspec]>=1.9.4
-gs = gcsfs==2021.11.0
+gs = gcsfs==2021.11.1
```

Comment:
While we are here, let's unpin gcsfs and ossfs and s3fs, since everything else is also unpinned anyway.